### DISCUSSION TITLE
Error saat melakukan step wise

### DISCUSSION THREAD

**Discussion by adrianaj1107** [teamalgoritma/community#125] [Q&A] [answered]
Halo, mau tanya, saat buat step wise, ada sperti ini :
![image](https://user-images.githubusercontent.com/104640746/171789558-cbb98a68-6346-4aff-a165-fe8ed391892d.png)

Mohon bantuannya.

Thank you,

**Comment by NabiilahArdini** [comment, marked as answer]
Halo @adrianaj1107 ,

Sepertinya error terjadi karena R salah menggunakan fungsi `step()`. Mungkin terjadi function conflict dari package yang digunakan (2 nama fungsi yang sama dari package yang berbeda). Unutk memastikan kita memakai fungsi yang tepat, boleh dipanggil `nama_package::nama_fungsi()`

Berikut contohnya:
```
model_step <- stats::step(object = model_all, direction = "backward", trace = F)
```

Semoga dapat membantu!
Bila jawaban ini dapat memantu Anda, boleh klik "Mark as Answer" 👍🏻

**Reply by adrianaj1107** [reply]
wahh bener..makasih banyak ya mentor...
